kythe/cxx/indexer/textproto/analyzer.cc

```diff
@@ -217,6 +236,121 @@ Status TextprotoAnalyzer::AnalyzeMessage(
   return OkStatus();
 }
 
+namespace {
```

Comment:
just put this in the extant anonymous namespace at the top of the file rather than opening a new one?

Reply:
Although you probably just want to use a (possible injected) https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.util.type_resolver for this instead.

Reply:
Removed the anonymous namespace - the existing anonymous namespace actually already includes this and most of the rest of the file.